### ISSUE TITLE
Memory usage is close to the limit.

### ISSUE THREAD

**Issue by hiqmatNisa** [josarajar/HTRTF#4]
Hi, 
thanks for this great work.
I am running this code on google colab on 12GB RAM. While loading the training set I am getting below error.
_Your runtime is close to its memory limit. Further use may cause an out-of-memory crash and the loss of in-memory state for all sessions. Would you like to terminate some sessions in order to free up memory (state will be lost for those sessions)? Currently, 10.62 GB / 12.72 GB is being used._
Can you please let us know about the running requirements(RAM, GPU) of this code? 
and how much time it takes for 250 epochs?

**Comment by josarajar** [comment]
Hi, 
Sorry for the delay, the requirements of RAM memory for the current batch size is of 15 GB. If you don't have enough RAM you can decrease the batch size.

Regarding the running time, in a Nvidia P100 the time for 250 epochs is about 4 days of training.
